src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { getRenderTarget, setIsEcamPage } from '../../../Common/defaults';
+import { SimVarProvider, useSimVar } from '../../../Common/simVars';
+import { PageTitle } from '../../Common/PageTitle';
+
+import './Apu.scss';
+
+setIsEcamPage('apu_page');
+
+export const ApuPage = () => {
+    // const gaugeOffset = -210;
+
+    const [apuAvail] = useSimVar('L:A32NX_OVHD_APU_START_PB_IS_AVAILABLE', 'Bool', 1000);
+
+    const [apuGenLoad] = useSimVar('L:A32NX_ELEC_APU_GEN_1_LOAD', 'Percent', 1000);
+    const [apuGenVoltage] = useSimVar('L:A32NX_ELEC_APU_GEN_1_POTENTIAL', 'Volts', 1000);
+    const [apuGenFreq] = useSimVar('L:A32NX_ELEC_APU_GEN_1_FREQUENCY', 'Hertz', 1000);
+
+    const [apuBleedOn] = useSimVar('L:A32NX_APU_BLEED_AIR_VALVE_OPEN', 'Bool', 1000);
+    const [apuBleedPressure] = useSimVar('L:APU_BLEED_PRESSURE', 'PSI', 1000);
+
+    // const [apuNValue] = useSimVar('L:A32NX_APU_N', 'Percent', 1000);
+    // const [apuEgtValue] = useSimVar('L:A32NX_APU_EGT', 'Celsius', 1000);
+
+    const [apuFlapOpenPercentage] = useSimVar('L:A32NX_APU_FLAP_OPEN_PERCENTAGE', 'Percent', 1000);
+    const [lowFuelPressure] = useSimVar('L:A32NX_APU_LOW_FUEL_PRESSURE_FAULT', 'Bool', 1000);
+
+    return (
+        <EcamPage name="main-apu">
+            <PageTitle x={283} y={33} text="APU" />
+
+            <ApuGenBox x={105} y={109} load={apuGenLoad} voltage={apuGenVoltage} freq={apuGenFreq} />
+            <AvailSig x={283} y={115} apuAvail={apuAvail} />
+            <BleedBox x={420} y={163} pressure={apuBleedPressure} apuBleedOn={apuBleedOn} />
+            <Divider x={83} y={227} />
+
+            <ApuFlapOpen x={400} y={375} apuFlapOpenPercentage={apuFlapOpenPercentage} />
+            <LowFuelPressureInd x={409} y={315} lowFuelPressure={lowFuelPressure} />
+        </EcamPage>
+    );
+};
+
+const SvgGroup = ({ x, y, children }) => <g transform={`translate(${x},${y})`}>{children}</g>;
```

Comment:
I would avoid creating components like this tbh, MSFS is sensitive to GC lag

Reply:
I think this is appropriate. We shouldn't micro-optimize this stuff away on a hunch. If this specific instance is an issue (and I don't think it is) we shouldn't be using React.

Reply:
I don't understand what advantage this brings, in terms of readability. Why do we need an `SvgGroup` component when `<g transform={`translate(${x},${y})`}>` is a few characters longer ? It's not about optimizing everything, it's rather than this particular abstraction seems a bit silly to me, abstracting away something which is perfectly readable in the first place, and potentially introducing a deeper object structure as a possible side effect.

Reply:
You've switched from arguing for performance in favor of readability. I don't have a strong opinion on what is more readable here, I can see your point. I was contesting the fact that we need to be super-mindful about every component we introduce, as that goes against the composability of React and can make working with it a lot more tedious.

Reply:
My point is that the potential and possible reduced burden on GC is worth it considering the little improvement in readability it brings.

Of course there are other factors to consider, such as setting `x={0) y={0}` effectively creates a Coherent layer which can reduce draw calls.

Reply:
I introduced this particular component in the ELEC page React rewrite, which brandoncabael is basing their work on. I'm big on componentisation of things and I like to abstract from higher-level to lower-level where the higher-level abstractions leak less of the HTML/SVG things. For me not having to repeat the same thing over and over again is a good thing. For this particular situation I can see you can argue the other way, however I do hope we're not removing the componentisation advantage of working in React. That would be a shame.

As beheh noted, if object creation really has a big performance impact, then we shouldn't be using React at all. It creates loads of objects.

Reply:
We could then add also a `layer` prop to this component to create GT layers.